dali/pipeline/operators/displacement/flip.h

```diff
@@ -30,14 +30,17 @@ class FlipAugment : public WarpAffineAugment {
   }
 
   void Prepare(Param* p, const OpSpec& spec, ArgumentWorkspace *ws, int index) {
-    float horizontal = (spec.GetArgument<int>("horizontal", ws, index)) ? -1.0 : 1.0;
-    float vertical = (spec.GetArgument<int>("vertical", ws, index)) ? -1.0 : 1.0;
-    p->matrix[0] = 1.0 * horizontal;
-    p->matrix[1] = 0.0;
-    p->matrix[2] = 0.0;
-    p->matrix[3] = 0.0;
-    p->matrix[4] = 1.0 * vertical;
-    p->matrix[5] = 0.0;
+    bool horizontal = 0 != spec.GetArgument<int>("horizontal", ws, index);
```

Comment:
I think `1 ==` would be make more sense here

Reply:
That would violate typical interpretation of integers as booleans (nonzero is true).